apps/web/src/presentation/components/launchpad/abc/ABCSettingsModelController.ts

```diff
@@ -32,6 +45,27 @@ export function useABCSettingsModelController(daoCreationRepository: DAOCreation
         init();
     }, []);
 
+    useEffect(() => {
+        async function checkBalance() {
+            const account = await getAccount();
+            const balance = await fetchBalance({
+                address: account.address as `0x${string}`,
+                token: daoCreationRepository.getDAOInfo().getABCConfig().collateralToken?.tokenAddress as `0x${string}`
+            });
+            const initialBalance = daoCreationRepository?.getDAOInfo()?.getABCConfig()?.getReserveInitialBalance() ?? 0
+            const parsedInitialBalance = parseEther(String(initialBalance));
+            if (initialBalance && balance.value >= parsedInitialBalance || initialBalance == 0) {
+                setEnoughBalance(true);
+            } else {
+                setEnoughBalance(false);
+            }   
+            console.log(enoughBalance);
+        }
+        checkBalance();
+    }, [
+        daoCreationRepository.getDAOInfo().getABCConfig().getReserveInitialBalance(), 
```

Comment:
Are we sure this is trigerring correctly?

Reply:
In my side is triggering without any problems (except for the other mentioned issue about the next button being disabled or not (errors showing properly)

Reply:
Already fixed - closing because outdated